### PR TITLE
Added logic to consider profiling enabled for few volumes

### PR DESCRIPTION
**Pull request by shtripat**
tendrl-bug-id: Tendrl/gluster-integration#553
Signed-off-by: Shubhendu <shtripat@redhat.com>

### CODE REVIEW

**Comment by shtripat** [comment]
@r0h4n @nthomas-redhat please review

**Comment by shtripat** [comment]
Verified the changes and if for few volumes profiling is enabled from CLI before import, the individual volumes are marked properly and cluster is marked profiling_enabled as `mixed`.

Need to verify the code with older version of glusterfs where `get-state` doesnt contain the profiling changes.

**Comment by shtripat** [comment]
@r0h4n @nthomas-redhat @a2batic @gnehapk plz review.
Need to add unit tests for additional flows and atoms.

File: tendrl/gluster_integration/objects/definition/gluster.yaml [hunk 1]
```diff
@@ -253,6 +253,12 @@ namespace.gluster:
         tags:
           help: optional parameters
           type: Dict
+        current_job:
```
Comment (r0h4n) [inline tendrl/gluster_integration/objects/definition/gluster.yaml]: why are these new attributes added to NativeEvent object? It should be for the Volume object

Comment (shtripat) [reply tendrl/gluster_integration/objects/definition/gluster.yaml]: oh thats a mistake. will correct.

File: tendrl/gluster_integration/objects/definition/gluster.yaml [hunk 2]
```diff
@@ -737,7 +737,61 @@ namespace.gluster:
...
+          type: Start
+          uuid: 242f6190-9b37-11e6-950d-a24fc0d96c05
+          help: Start volume profling
+        StopProfiling:
+          enabled: true
+          inputs:
+            mandatory:
+              - Volume.vol_id
+              - Volume.volname
+          name: delete_volume
```
Comment (r0h4n) [inline tendrl/gluster_integration/objects/definition/gluster.yaml]: why the name "delete_volume" ?

Comment (shtripat) [reply tendrl/gluster_integration/objects/definition/gluster.yaml]: Ctrl C + Ctrl V issue :) will correct.

**Comment by shtripat** [comment]
>> Verified?

Yes

**Pull request merged**